cli/src/commands/auth/commands/logout.ts

```diff
@@ -0,0 +1,21 @@
+import { existsSync, rmSync } from 'node:fs';
+import { Command } from 'commander';
+import pc from 'picocolors';
+import { configDir, configFile } from '../../../core/config.js';
+import program from '../../index.js';
+import { BaseCommandOptions } from '../../../core/types/types.js';
+
+export default (opts: BaseCommandOptions) => {
+  const logoutCommand = new Command('logout');
+  logoutCommand.description('Logout a user.');
+
+  logoutCommand.action(() => {
+    if (!existsSync(configFile)) {
```

Comment:
I think we can skip this. Logout can be idempotent.

Reply:
Done